data/registry/application-integration-WSO2-Kubernetes-Gateway.yml

```
@@ -0,0 +1,18 @@
+title: WSO2 Kubernetes Gateway
+registryType: application integration
+language: Java
+tags:
+  - WSO2
+  - API Gateway For Kubernetes
+  - Java
+  - Opentelemetry
+license: Apache-2.0
+description: 'Integration of OpenTelemetry with WSO2 Kubernetes Gateway(WSO2 APK Gateway) enhances observability by providing insights into API metrics, traces, and logs, facilitating better performance monitoring and troubleshooting within Kubernetes environments.'
+authors:
+  - name: WSO2
+    url: https://github.com/wso2
+urls:
+  website: https://wso2.com/api-management/
+  docs: https://apk.docs.wso2.com/en/latest/administration/distributed-tracing/#opentelemetry-protocol-otlp
```

```suggestion
urls:
  website: https://wso2.com/api-management/
  docs: https://apk.docs.wso2.com/en/latest/administration/distributed-tracing/#opentelemetry-protocol-otlp
  repo:
```

Since this is open source, please add the repo to the related project